xml/System.IO/DirectoryInfo.xml

```diff
@@ -250,6 +250,8 @@
  If the directory already exists, this method does nothing.  
 
  If the directory did not exist before calling this method, then any cached attribute information about the directory will be flushed if the creation is successful.  
+
+ If the directory path contains non-existent intermediate directories they will be created also
```

Comment:
We already have a wording for this in DirectoryInfo.CreateSubdirectory(String) 
> Any and all directories specified in path are created, unless some part of path is invalid. The path parameter specifies a directory path, not a file path. If the subdirectory already exists, this method does nothing.

and Directory.CreateDirectory(String)
> Any and all directories specified in path are created, unless they already exist or unless some part of path is invalid. If the directory already exists, this method does not create a new directory, but it returns a [DirectoryInfo](https://learn.microsoft.com/en-us/dotnet/api/system.io.directoryinfo?view=net-6.0) object for the existing directory.


This description should be aligned to those two descriptions of the same.